.env.dev

```diff
@@ -22,3 +20,7 @@
 #MAILER_DSN=null://null
 
 #ILIOS_TIKA_URL=null
+
+###> symfony/framework-bundle ###
+ILIOS_SECRET=35d19d470bf1b44e82f84e8c74e149d8
```

Comment:
is this addition intentional?

Reply:
Yes. The framework bundle now auto generates a fake secret to go in this spot (`App_SECRET`). I'm not sure if it has value or now, but I left it in until we know otherwise.